src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Chaining_NoFlag.txt

```diff
@@ -0,0 +1,2 @@
+﻿>> 3;
+Errors: Error 1-2: The chaining operator is only allowed in a behavior property. Make sure that EnableExpressionChaining flag is properly set and passed as argument.|Error 1-2: Expected operator. We expect an operator such as +, *, or & at this point in the formula.|Error 2-2: Expected an operand. The formula or expression expects a valid operand. For example, you can add the operand '2' to the expression ' 1 +_' so that the result is '3'. Or, you can add the operand "there" to the expression '"Hi "& _ ' so that the result is 'Hi there'.|Error 1-2: Expected operator. We expect an operator such as +, *, or & at this point in the formula.
```

Comment:
>Make sure that EnableExpressionChaining flag is properly set and passed as argument. [](http://example.com/codeflow?start=81&length=84)

We should probably exclude this. Different audiences: because these errors messages are for end-users, and the flag is for SDK consumers.

Reply:
Can we avoid the rest of the error message too?  It becomes confusing after the initial message.